qemu/tests/cfg/vm_poisoned_migration_prevention.cfg

```diff
@@ -0,0 +1,15 @@
+- vm_poisoned_migration_prevention:
+    only Linux
+    required_qemu = [9.0.0,)
+    type = vm_poisoned_migration_prevention
+    virt_test_type = qemu
+    smp = 8
+    vcpu_maxcpus = ${smp}
+    slots_mem = 4
+    maxmem_mem = 32G
+    mem = 4096
+    kill_vm_on_error = yes
+    mig_timeout = 1200
+    migration_protocol = "tcp"
+    hwpoison_file = ""
```

Comment:
Suggest add a comment above to explain user need provide it to run this test.

Reply:
Updated !